server/views/account.html

```diff
@@ -7,7 +7,14 @@ <h2 class="govuk-heading-l">Account</h2>
   <p>Email: <strong>{{ user.id }}</strong></p>
   <p>Office Location: <strong>{{ user.officeLocation }}</strong></p>
   <p>Organisation: <strong>{{ user.orgName }}</strong></p>
-  <p>Role(s): <strong>{{ data.roles | sort() | join(', ') }}</strong></p>
+  <p>{{ 'Role' if data.roles.length == 1 else 'Roles' }}: <strong>{{ data.roles | sort() | join(', ') }}</strong></p>
+
+  {{
+    govukDetails({
+      summaryText: "Help with account",
+      html: "<strong>Office location</strong> and <strong>organisation</strong> are sourced from the Azure Active Directory (AAD) <a href='https://portal.office.com/account/#personalinfo'>profile</a>. If the raw value of either can not be matched to a known value it will be defaulted to <strong>unassigned</strong> and <strong>undefined</strong>, respectively.<br>This will not inherently prevent messages from being sent or received. However, in order to help with accurate message distribution both office location and organisation should be up to date."
```

Comment:
Trying to provide some useful info for people when they see `undefined` and `unassigned` which is a bit strange if you don't know what it means.